DAAppsViewController/DAAppViewCell.m

```diff
@@ -12,6 +12,25 @@
 static NSArray *_starRatingImages = nil;
 static NSNumberFormatter *_decimalNumberFormatter = nil;
 
+@interface UIImage (DAAppViewCell)
++ (UIImage *)imageNamedFromMainBundleOrFramework:(NSString *)name;
+@end
+
+@implementation UIImage (DAAppViewCell)
+
++ (UIImage *)imageNamedFromMainBundleOrFramework:(NSString *)name
+{
+    UIImage *image = [UIImage imageNamed:name];
+    if (!image) {
+        // Try to load from bundle
+        NSBundle *bundle = [NSBundle bundleForClass:[DAAppViewCell class]];
+        image = [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
```

Comment:
This is only available in iOS 8.0 or newer. This pod specifies min target of iOS 5.0 in the readme, but the podspec doesn't actually specify a min target.